selftests/unit/test_utils_partition.py

```diff
@@ -45,12 +37,8 @@ class TestPartition(unittest.TestCase):
 
     @unittest.skipIf(missing_binary('mkfs.ext2'),
                      "mkfs.ext2 is required for these tests to run.")
-    @unittest.skipIf(missing_binary('sudo'),
+    @unittest.skipIf(not process.can_sudo(),
```

Comment:
I'm trying to say that running `mount` without sudo binary passes, because if avocado fails to find `sudo` it simply goes without it and `mount` works fine for non-root users.

Yes, the `mount` check would be better for such setups, so how about this. We should check whether `sudo` binary is available and then try to execute the sudoed command?